hub/05-view-device-data.md

```diff
@@ -0,0 +1,92 @@
+
+``` bash
+hzn version
+```
+
+It should respond with something similar to this:
+
+``` bash
+Horizon CLI version: 2.25.0
+Horizon Agent version: 2.25.0
+```
+
+Check to ensure that the `edgex_1.0.1_amd64` Service is running and your Edge Node is properly configured:
+
+``` bash
+hzn node list | jq '.configstate'
+```
+
+You should see output similar to this:
+
+``` bash
+{
+  "state": "configured",
+  "last_update_time": "2020-04-04 07:37:39 -0700 PDT"
+}
+```
+
+And last, let's ask Docker to show us the running images:
+
+``` bash 
+docker ps
+```
+
+Which should return something like this:
+
+``` bash
+CONTAINER ID        IMAGE                                    COMMAND                  CREATED             STATUS              PORTS                                              NAMES
+4b764ec9140c        edgexfoundry/docker-core-metadata-go     "/core-metadata --re…"   6 hours ago         Up 6 hours          0.0.0.0:48081->48081/tcp, 48082/tcp                1a85b4ce42a70ca27b0d1eee80c154e56ca8dda50f41199ccb4ec5e253a9c1cd-edgex-core-metadata
+1c1b59fe238c        edgexfoundry/docker-core-data-go         "/core-data --regist…"   6 hours ago         Up 6 hours          0.0.0.0:5563->5563/tcp, 0.0.0.0:48080->48080/tcp   1a85b4ce42a70ca27b0d1eee80c154e56ca8dda50f41199ccb4ec5e253a9c1cd-edgex-core-data
+d88ec30ecb59        edgexfoundry/docker-core-command-go      "/core-command --reg…"   6 hours ago         Up 6 hours          0.0.0.0:48082->48082/tcp                           1a85b4ce42a70ca27b0d1eee80c154e56ca8dda50f41199ccb4ec5e253a9c1cd-edgex-core-command
+8803a65bb1de        edgexfoundry/docker-support-logging-go   "/support-logging --…"   6 hours ago         Up 6 hours          0.0.0.0:48061->48061/tcp                           1a85b4ce42a70ca27b0d1eee80c154e56ca8dda50f41199ccb4ec5e253a9c1cd-edgex-support-logging
+e3c0775618c7        edgexfoundry/docker-edgex-mongo          "docker-entrypoint.s…"   6 hours ago         Up 6 hours          0.0.0.0:27017->27017/tcp                           1a85b4ce42a70ca27b0d1eee80c154e56ca8dda50f41199ccb4ec5e253a9c1cd-edgex-mongo
+af4e5e061f49        edgexfoundry/docker-edgex-volume         "/bin/sh -c '/usr/bi…"   6 hours ago         Up 6 hours                                                             1a85b4ce42a70ca27b0d1eee80c154e56ca8dda50f41199ccb4ec5e253a9c1cd-edgex-files
+a9517da05d16        edgexfoundry/docker-device-random-go     "/device-random --pr…"   6 hours ago         Up 6 hours          0.0.0.0:49988->49988/tcp                           1a85b4ce42a70ca27b0d1eee80c154e56ca8dda50f41199ccb4ec5e253a9c1cd-edgex-device-random
+```
+
+Verify that you see seven images, and that CREATED and STATUS columns show that none of them are restarting.
+
+## View the data
+
+According to the documentation, you could request the URL for Core Data, but that will show you all of the events and readings since the Service was started, which could get quite long:
```

Comment:
You're switching context from horizon to edgex at this point, I think a statement to this effect would be helpful.

Reply:
Very good point.  Adding.